src/components/atoms/Button/index.ts

```diff
@@ -0,0 +1,3 @@
+import Button from './Component';
+
+export { Button };
```

Comment:
@jhonangelo Add an empty new line on the last part of a file. All files should have an empty line as the last line so that there will be no error/warnings on github

Reply:
@fctmolina1135 Is this applicable for all files or just for index files?

Reply:
@jhonangelo This is applicable for all files